### ISSUE TITLE
Accessibility testing software

### ISSUE THREAD

**Issue by gestchild** [wellcomecollection/wellcomecollection.org#10007]
The [Practical Accessibility course](https://practical-accessibility.today/) details the [most popular screen reader/browser combinations we should test on](https://practical-accessibility.today/chapters/testing-environment-setup/#which-browser-and-screen-reader-pairings-should-you-test-on%3F).

We should get access to these for developers and any one else who needs them and set them up as recommended in the course.

We need to be able to test:
Desktop:
- [ ] Jaws with Chrome on Windows - available on AssistivLabs once license bought
- [x] NVDA with Firefox on Windows - available on AssistivLabs/Browserstack
- [x] Narrator with Edge on Windows - available on AssistiveLabs
- [x] Voiceover with Safari on Mac - available on Browserstack

Mobile:
- [ ] Voiceover with Safari on iOS - coming to Browserstack (limited testing available with xcode Simulator and Accessibility Inspector - see below)
- [x] Google Talkback with Chrome on Android - available on Browserstack

Screen Reader User Survey:
[Primary Screen Reader](https://webaim.org/projects/screenreadersurvey9/#primary)
[Screen Readers Commonly Used](https://webaim.org/projects/screenreadersurvey9/#used)

~N.B. There is already a virtual machine (WTV000007) set up running a demo version of Jaws, that more people should have access to.~

**Comment by gestchild** [comment]
There is a ticket open with IT Service desk about getting access to the virtual machine and installing additional software; Number: REQ0037821

**Comment by gestchild** [comment]
We are able to access these through Browserstack:

- NVDA with Firefox on Windows
- Google Talkback with Chrome on Android
- Voiceover with Safari on Mac (also on our own computers)

This is coming to Browserstack:

- Voiceover with Safari on iOS

These are accessible through remote desktop:
 - NVDA with Firefox on Windows
 - Jaws with Chrome on Windows

However, @gestchild is still the only person able to access it. Also, there are issues with running the trial version of JAWS at the moment.

I'm going to experiment with running a VM on my local machine.

**Comment by gestchild** [comment]
Closing this - the only outstanding task is to work out licensing for JAWS - see #10425
